packaging/torchvision/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - ffmpeg >=4.2  # [not win]
     # NOTE: Pinned to fix issues with size_t on Windows
     - jpeg <=9b
-    - pillow >=5.3.0
```

Comment:
Pillow was pinned to `>=5.3.0` and code was refactored accordingly. #3641 to fix #3631.

I'm not very proficient with packaging in torchvision though.

Reply:
Thanks for looking into it! Any ideas why here it's pinned to 5.3 while above to 4.1.1? 

Do we still need pinning or we can remove it from everywhere as I did here? If we still need it, I'm happy to update the PR to have everywhere the same pinned version. My target is to be consistent across all files.

Reply:
Here is a [comment](https://github.com/pytorch/vision/pull/3638#issuecomment-814274883) that summarizes this

In short, pillow draws rectangles / boxes slightly different in different versions. 
Also Pillow 4.1. is slightly old and some APIs which we use from pillow need 5.3, so it was decided to bump the minimum version to 5.3

This causes some tests to be flaky, to fix this pillow was pinned.

Also marking #3647

cc @NicolasHug

Reply:
Thanks for providing more context. I agree with the remarks of the linked comment that we should use one pinned version not multiple. I've updated the PR to pin 5.3 but more changes might be necessary. Examples:

Unclear why whether we should use >= here:
https://github.com/pytorch/vision/blob/a64b54acdb5e7afea93b9cedf08c55137b05098a/.circleci/unittest/windows/scripts/install.sh#L34

Unclear if we should pin in places such as:
https://github.com/pytorch/vision/blob/3782888352b405441a631baa962a7bbb5a839fa8/.circleci/config.yml#L539

And if we do pin 5.3 everywhere, shall we remove from documentation notes like this?
https://github.com/pytorch/vision/blob/2f3b7ca971c3a980833c8318fc1f0c8fe83ef8d2/torchvision/transforms/transforms.py#L700

I recommend not merging this until we get input from most relevant people and cover the above questions.

Reply:
> And if we do pin 5.3 everywhere, shall we remove from documentation notes like this?

Yes, it looks like I missed a few of these when I did https://github.com/pytorch/vision/pull/3641, sorry. We should mention and use pil >= 5.3.0 everywhere I think. For more context: having a dependency with multiple minimum required version can be huuuge headache for users (https://github.com/pytorch/vision/pull/3638#issuecomment-814274883) so it's best to be consistent everywhere

Reply:
I will follow up with a PR cleaning documentations / code changes for Pillow < 5.3.0. Most is cleaned I believe but for sanity check.

Reply:
@oke-aditya Don't worry I'm going to update this PR with that soon. I would really appreciate a thorough review to confirm I did not miss a spot. :)